### ISSUE TITLE
Duplicates in downloaded files

### ISSUE THREAD

**Issue by queencaffeine** [queencaffeine/Civitai-Image-Downloader#1]
TL;DR - version 1 doesn't work as intended, version 1.1 incoming.

I noticed a fatal flaw in this when I ran it which I hadn't noticed beforehand - the same 300 or so images were downloaded over and over again instead of downloading the full number of images. I'm currently testing a new version of the code which should rectify that issue and it will be released and updated here once I can verify it's successfully downloaded my 3,500 images without duplicates.

If you have used the current version (v1) please check your downloads and remove any duplicates and/or when v1.1 is released, use that instead to download all your images. If you still have duplicates on v1.1 please contact me and I will continue fixing it until it is working.
